pkg/operator/minio/controller.go

```diff
@@ -110,34 +108,42 @@ func (c *MinioController) makeMinioHeadlessService(name, namespace string, spec
 	return svc, err
 }
 
-func (c *MinioController) buildMinioCtrArgs(statefulSetPrefix, headlessServiceName, namespace, clusterDomain string, serverCount int32) []string {
-
+func (c *MinioController) buildMinioCtrArgs(statefulSetPrefix, headlessServiceName, namespace, clusterDomain string, serverCount int32, volumeClaims []v1.PersistentVolumeClaim) []string {
 	args := []string{"server"}
 	for i := int32(0); i < serverCount; i++ {
-		args = append(args, makeServerAddress(statefulSetPrefix, headlessServiceName, namespace, clusterDomain, i))
+		for k := range volumeClaims {
```

Comment:
what happens if no volumeClaims are included in the spec?

Reply:
Fixed. An emptyDir is now used when no volume claims are given.